web/client/main.js

```diff
@@ -288,7 +288,7 @@ function setupChat() {
         var messageNode = document.createElement('div');
         var messageContent = document.createElement('div');
         messageNode.classList.add('chatMessage');
-        messageContent.innerHTML = msg;
+        messageContent.textContent = msg;
```

Comment:
Turns out the bug was quite serious user could inject client-side script such as:
<p style="color:red">XSS vulnerability</p>